test/Kestrel.Core.Tests/Http2ConnectionTests.cs

```diff
@@ -2161,6 +2208,10 @@ public Task HEADERS_Received_HeaderBlockContainsTEHeader_ValueIsNotTrailers_Stre
 
             await SendGoAwayAsync();
 
+            await SendRstStreamAsync(1);
+            await SendRstStreamAsync(3);
+            await SendRstStreamAsync(5);
```

Comment:
The point of this test is that a connection-level abort _by itself_ is enough to complete any calls to Response.Body.WriteAsync that were waiting on the connection output flow control window to update that. This is no longer testing that with the extra stream reset frames.

Reply:
Sending a GOAWAY isn't by itself enough enough to signify a shutdown of an Http2 connection. By the phrasing in the RFC, it's a signal that there's an intention to shutdown the connection in the near future. A connection-level abort seems to be simply closing the transport-level connection. Should I simulate that here instead?

Reply:
Yeah, close the pipe.